protocol/messenger_handler.go

```diff
@@ -213,6 +215,15 @@ func (m *Messenger) HandleMembershipUpdate(messageState *ReceivedMessageState, c
 	return nil
 }
 
+func (m *Messenger) checkIfCreatorIsOurContact(group *v1protocol.Group) bool {
+	creator, err := group.Creator()
+	if err == nil {
+		contact, _ := m.allContacts.Load(creator)
```

Comment:
You need to check the error here as well in case

Reply:
just return false when get err from group.Creator() was what i expected, i've added a logger for this error, passing this error up to HandleMembershipUpdate may will make things complicated, WDYT of the latest commit? @jrainville

Reply:
Ah, I see that you check if `contact` is `nil`. That's fine then.